modules/ROOT/pages/index.adoc

```diff
@@ -3,12 +3,76 @@
+
+++++
+        <p>By installing the SDK, you agree to our <a href="https://sdk.dfinity.org/sdk-license-agreement.txt">Terms & Conditions</a>.</p>
+        </div>
+    </div>
+
+<div class="home-pitches-grid">
+
+    <div class="landing-pitch">
+        <img src="_images/HelloWorld.png" height="320" width="320" />
+        <h2>What can you imagine?</h2>
+        <p>Decentralized software can be more than financial transactions. The Internet Computer is a general purpose platform and our developer community has built everything from video chat to mobile games to compilers.</p>
+        <p class="link"><a href="http://github.com/dfinity/awesome-dfinity">Check out some awesome apps</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/CorporateCloud.png" height="320" width="320" />
+        <h2>Escape the Corporate Cloud</h2>
+        <p>Why build decentralized software just to run your front-end on monopolistic hardware like Amazon Web Services or Google Cloud? Build your front-end in any framework (React, Angular, Vue, or even vanilla JavaScript) and your users can access it from a standard browser — <b>no plug-ins</b>.</p>
+        <p class="link"><a href="https://sdk.dfinity.org/docs/developers-guide/webpack-config.html">Read more about front-end development</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/MotokoLogo.png" height="320" width="320" />
+        <h2>Your new favorite language</h2>
+        <p>The <b>Motoko</b> language is optimized for authoring for the Internet Computer and packed with so many familiar features you may forget you're working with a blockchain: async/await, randomness, time and date, and even floating point numbers. </p>
+        <p class="link"><a href="https://sdk.dfinity.org/docs/language-guide/motoko.html">Read more about Motoko</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/RustLogo.png" height="320" width="320" />
+        <h2>On the shoulders of giants</h2>
+        <p>Thanks to the interface description language <b>Candid</b>, every service running on the Internet Computer is interoperable with every other (regardless of programming language). That means you can import external services as easily as libraries and extend the Open Source revolution with Open Internet Services.</p>
+        <p class="link"><a href="https://sdk.dfinity.org/docs/candid-spec/idl">Read more about Candid</a></p>
+    </div>
+
+    <div class="landing-pitch">
+        <img src="_images/DeleteDatabase.png" height="320" width="320" />
+        <h2>Go ahead, delete your database</h2>
+        <p>The radical rethinking baked into the Internet Computer means much of the typical complexity of software architecture melts away. You won't need a firewall, a VPN, usernames and passwords — and with <b>Orthogonal Persistence</b>, any data you create in your canister is automatically persisted in memory pages making databases a thing of the past.</p>
```

Comment:
maybe use **application** instead of canister in the description since we can't assume the landing page viewer has any idea what we mean by canister.